res/css/photon-components.css

```diff
@@ -48,3 +50,30 @@
 .photon-button-ghost:hover:active {
   background-color: var(--grey-90-a20);
 }
+
+/* --- Inputs --- */
+/* See https://design.firefox.com/photon/components/input-fields.html for the spec */
+.photon-input {
+  box-shadow: none; /* remove the default Firefox style */
+  border: 1px solid var(--grey-90-a20);
+  border-radius: 2px;
+  padding: 0 8px;
+  height: 32px;
+}
+
+.photon-input:hover {
+  border-color: var(--grey-90-a30);
+}
+
+.photon-input:invalid:not(:placeholder-shown) {
+  box-shadow: 0 0 0 1px var(--red-60), 0 0 0 4px rgba(251, 0, 34, 0.3);
```

Comment:
Fired the PR https://github.com/FirefoxUX/design-tokens/pull/56 to have a variable for the rgba constant here.